src/supporting-data/citations/components/results/CitationCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card } from 'franklin-sites';
+import { FC, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { getEntryPath } from '../../../../app/config/urls';
+import { getIdKeyFor } from '../../../../shared/components/results/ResultsView';
+import { Namespace } from '../../../../shared/types/namespaces';
+import { CitationsAPIModel } from '../../adapters/citationsConverter';
+import LiteratureCitation from '../LiteratureCitation';
+
+const CitationCard: FC<{
+  data: CitationsAPIModel;
+  selected?: boolean;
+  handleEntrySelection?: (rowId: string) => void;
+}> = ({ data, selected, handleEntrySelection }) => {
+  const history = useHistory();
+
+  const key = getIdKeyFor(Namespace.citations)(data);
```

Comment:
If you're going to use the curried version, do `const getIdKey = getIdKeyFor(Namespace.citations)` outside this component, and do `const key = getIdKey(data);`